mode/css/css.js

```diff
@@ -336,9 +336,9 @@ CodeMirror.defineMode("css", function(config, parserConfig) {
   return {
     startState: function(base) {
       return {tokenize: null,
-              state: "top",
+              state: config.inline ? "block" : "top",
```

Comment:
I think it makes more sense to put this on the `parserConfig` object -- that's where mode-specific options go.